src/components/views/styl/SearchBox.styl

```diff
@@ -0,0 +1,23 @@
+@import 'includes/variables';
+
+.search-box {
+    border-radius: 10px;
+    border: 1px solid $gray;
+    display: inline-flex;
+    padding: 3px;
+
+    &::after {
+        color: $gray;
+        content: '\f002';
+        font-family: 'FontAwesome';
+
+        background: #fff;
+        padding: 3px;
+        z-index: 2;
```

Comment:
I prefer z-indexes that are 10-based (in case we want to inject things between layers later) though this is ultimately your call - I think the rest of this project increments by 1.

Reply:
That makes sense. I'll take any strategy for z-indices, I never know what to do. :smile: